test/k8sT/Policies.go

```diff
@@ -827,78 +736,157 @@ EOF`, k, v)
 		})
 	})
 
-	Context("KubernetesNetworkPolicy between server and client", func() {
+	Context("Namespaces policies", func() {
 
 		var (
-			policy = "allow-ns-b-via-namespace-selector"
+			err         error
+			ciliumPod   string
+			namespace   = "second"
+			appPods     map[string]string
+			appPodsNS   map[string]string
+			clusterIP   string
+			clusterIPNS string
+
+			demoPath         = helpers.ManifestGet("demo.yaml")
+			l3Policy         = helpers.ManifestGet("l3_l4_policy.yaml")
+			netpolNsSelector = fmt.Sprintf("%s -n %s", helpers.ManifestGet("netpol-namespace-selector.yaml"), namespace)
+			policy           = fmt.Sprintf("%s -n %s", l3Policy, namespace)
```

Comment:
Rename this variable. "policy" is quite vague. Please try to make the variable more expressive of what type of policy it represents.